src/composables/emailPasswordValidator.js

```diff
@@ -0,0 +1,16 @@
+function emailPasswordValidator(email, password) {
+  var regxEmail = /^([a-zA-Z0-9\.]+)@([a-zA-Z]+)\.([a-z]+)(\.[a-z]+)?$/;
+
+  var regxPassword =
+    /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{8,15}$/;
+
+  regxEmail.test(email)
+    ? console.log("Valid email address!")
```

Comment:
We don't need to have a console.log

We could just return an error message to the user